src/basketball/worker/core/GameSim.js

```diff
@@ -1104,10 +1107,15 @@ class GameSim {
 				probMake += 0.1;
 			}
 		}
-
-		let foulFactor =
-			0.65 *
-			(this.team[this.o].player[p].compositeRating.drawingFouls / 0.5) ** 2;
+		let foulFactor;
+		if (g.foulRate > 0) {
```

Comment:
Does it really matter if it's less than 0? Wouldn't it behave the same as 0?

If it doesn't matter, then delete the conditional check here. If it does matter, shouldn't you also have this condition above for non-shooting fouls too?